scene/resources/bit_map.cpp

```diff
@@ -505,17 +593,18 @@ Vector<Vector<Vector2>> BitMap::clip_opaque_to_polygons(const Rect2i &p_rect, fl
 			if (!fill->get_bit(j, i) && get_bit(j, i)) {
 				fill_bits(this, fill, Point2i(j, i), r);
 
-				Vector<Vector2> polygon = _march_square(r, Point2i(j, i));
-				print_verbose("BitMap: Pre reduce: " + itos(polygon.size()));
-				polygon = reduce(polygon, r, p_epsilon);
-				print_verbose("BitMap: Post reduce: " + itos(polygon.size()));
+				for (Vector<Vector2> polygon : _march_square(r, Point2i(j, i))) {
+					print_verbose("BitMap: Pre reduce: " + itos(polygon.size()));
```

Comment:
I wouldn't keep those print_verbose statements here. Print verbose should be used to provide more detailed info about some process, but this will just result in console spam without much value for the user.

Reply:
Done! Removed the one for `fill_bits` too for the same reason